go/storage/mkvs/db/badger/migrate.go

```diff
@@ -665,15 +681,647 @@ func (v4 *v4Migrator) Migrate() (rversion uint64, rerr error) {
 		}
 	}
 
-	// All done, flush and clean up. The metadata blob will be removed
-	// in the defer handler.
+	// All done, flush and clean up.
 	v4.done = true
 	if err := v4.flush(true); err != nil {
 		return 0, err
 	}
 	return 4, nil
 }
```

Comment:
Can you please also update v4 migration to log something after its done migrating. iirc it doesn't explicitly log anything when everything is migrated.

Reply:
Done.